app/validators/length.js

```diff
@@ -0,0 +1,6 @@
+export default function lengthValidator() {
+  return (key, oldValue) => {
+    let length = Object.keys(oldValue).length
```

Comment:
instead of calling hte variable, "length", call it something more description like, "standardsCount" or "numberOfStandards"